src/containers/Tenant/utils/schemaActions.tsx

```diff
@@ -324,6 +329,7 @@ export const getActions =
 
             table: ROW_TABLE_SET,
             column_table: COLUMN_TABLE_SET,
+            system_table: SYSTEM_TABLE_SET,
```

Comment:
Support new type added in new `ydb-ui-components` version, full system tables support will be in a separate PR